.buildkite/release-automation/config.yml

```diff
@@ -13,6 +13,7 @@ runner_queues:
   macos: macos
   macos-arm64: macos-pr-arm64
   windows: windows_queue_pr
+  small_branch: runner_queue_small_branch
```

Comment:
could you split this config change into a separate PR?

Reply:
done! PR is #43519